pinot-controller/src/test/java/org/apache/pinot/controller/validation/StorageQuotaCheckerTest.java

```diff
@@ -104,33 +105,39 @@ public void testWithinQuota()
     // No response from server, should pass without updating metrics
     mockTableSizeResult(-1, 0);
     assertTrue(isSegmentWithinQuota());
-    assertEquals(
-        controllerMetrics.getValueOfTableGauge(OFFLINE_TABLE_NAME, ControllerGauge.OFFLINE_TABLE_ESTIMATED_SIZE), 0);
+    assertFalse(
+        MetricValueUtils.gaugeExists(controllerMetrics,
```

Comment:
similar comment here: should we have a MetricValueUtils.tableGaugeExists() than pass name of table as an argument?